persistence-elasticsearch/core/src/main/java/org/apache/unomi/persistence/elasticsearch/ElasticSearchPersistenceServiceImpl.java

```diff
@@ -139,8 +139,10 @@
     private String indexPrefix;
     private String monthlyIndexNumberOfShards;
     private String monthlyIndexNumberOfReplicas;
+    private String monthlyIndexMappingTotalFieldsLimit;
     private String numberOfShards;
     private String numberOfReplicas;
+    private String indexMappingTotalFieldsLimit;
```

Comment:
Why you didn't add this property to the internal create index mapping?

Reply:
It was a mistake. I added it (with a bad copy-paste) but I committed your fix. Thanks for catching this (twice !)